File: src/main/java/example/refreshtokens/apollo/Application.java
package example.refreshtokens.apollo;

import com.spotify.apollo.Environment;
import com.spotify.apollo.httpservice.HttpService;
import com.spotify.apollo.httpservice.LoadingException;
import example.refreshtokens.apollo.controller.Routes;

public class Application {


    public static void main(String... args) throws LoadingException {
        HttpService.boot(Application::init, "refreshtokenexample", args);
    }

    static void init(Environment environment) {
        environment.routingEngine()
                .registerAutoRoutes(new Routes());
    }
}

File: src/main/kotlin/example/refreshtokens/apollo/Application.kt
package example.refreshtokens.apollo

import com.spotify.apollo.*
import com.spotify.apollo.httpservice.HttpService
import example.refreshtokens.apollo.controller.Routes

object ApplicationKotlin {

    @JvmStatic
    fun main(args: Array<String>) {
        HttpService.boot(AppInit { init(it) }, "refreshtokenexample", *args)
    }

    fun init(environment: Environment) {
        environment.routingEngine()
                .registerAutoRoutes(Routes())
    }
}

